test/tokens/ERC20.t.sol

```diff
@@ -53,6 +53,7 @@ contract ERC20Test is Test {
 
         // Deploy the Mintable ERC20
         address mintableTokenAddress = HuffDeployer.config()
+            .with_evm_version("shanghai")
```

Comment:
it seems to be the default ([src](https://github.com/huff-language/foundry-huff/blob/bcac40d5b0950588b30cdf24dff25df080007295/src/HuffConfig.sol#L140)). You want to do this to avoid future issues ?